### ISSUE TITLE
cannot generate event description

### ISSUE THREAD

**Issue by BuptStEve** [HcySunYang/vuese#37]
**Describe the bug**

```vue
<script>
export default {
    created () {
        const a = { a: 1 }
        
        // foobar
        this.$emit('select', a)
    },
}
</script>
```

```md
# [name]

## Events

<!-- @vuese:[name]:events:start -->
|Event Name|Description|Parameters|
|---|---|---|
|select|-|-|

<!-- @vuese:[name]:events:end -->
```

**To Reproduce**

1. 代码如上，采用 standard 标准不写分号
2. 在 `$emit` 上是对象时无法生成 description
3. 添加分号，或者添加括号，或者空对象则可以生成 description

**Expected behavior**

希望能生成 description

**Which version do you use:**

`@vuese/parser: 2.0.1`

**Comment by HcySunYang** [comment]
This is the behavior of the babel parser, and which node the annotation belongs to varies according to the semantics. But this is really a bug.

**Comment by HcySunYang** [comment]
I just released a patch version of 2.0.2, which solves this problem.
